content/en/docs/tasks/policy-enforcement/rate-limit/index.md

```diff
@@ -77,7 +80,7 @@ A [reference implementation](https://github.com/envoyproxy/ratelimit) of the API
 1. Apply an `EnvoyFilter` to the `ingressgateway` to enable global rate limiting using Envoy's global rate limit filter.
 
     The patch inserts the
-    `envoy.filters.http.ratelimit` [global envoy filter](https://www.envoyproxy.io/docs/envoy/latest/api-v3/extensions/filters/http/ratelimit/v3/rate_limit.proto#envoy-v3-api-msg-extensions-filters-http-ratelimit-v3-ratelimit) filter into the `HTTP_FILTER` chain.
+    `envoy.filters.http.ratelimit` [global envoy filter](https://www.envoyproxy.io/docs/envoy/latest/api-v3/extensions/filters/http/ratelimit/v3/rate_limit.proto#envoy-v3-api-msg-extensions-filters-http-ratelimit-v3-ratelimit) into the `HTTP_FILTER` chain.
```

Comment:
If you shortened the previous lines prior to the [, let's move the [...] to the prior lone so this line is shorter.

Reply:
Thanks, done.